### ISSUE TITLE
Column names containing "/" fail on some calls

### ISSUE THREAD

**Issue by bdemchak** [cytoscape/py4cytoscape#42]
When a column name is used as part of a URL in a CyREST call, and the column name contains "/", the URL is mis-formed. 

To see this, put "/" in the name of a table column, and then execute get_table_columns().

**Comment by bdemchak** [comment]
Similar bug exists in RCy3: https://github.com/cytoscape/RCy3/issues/108

**Comment by AlexanderPico** [comment]
Yep. This commit fixed other column name character issues, but `/` remains a problem.
https://github.com/cytoscape/RCy3/commit/39fd09f127a6054e077c624044a8e5cd5ce06232
